docs/release-notes/0.2/release-0.2.md

```diff
@@ -0,0 +1,61 @@
+# ML.NET 0.2 Release Notes
+
+We would like to thank the community for the engagement so far and helping us shape ML.NET.
+
+Today we are releasing ML.NET 0.2. This release focuses on addressing questions/issues, adding clustering to the list of supported machine learning tasks, enabling using data from memory to train models, easier model validation, and more.
+
+### Installation
+
+Supported platforms: Windows, MacOS, Linux (see [supported OS versions of .NET Core 2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) for more detail)
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
+```
+
+### Release Notes
+
+Below are some of the highlights from this release.
+
+#### New machine learning tasks: clustering
+
+Clustering is an unsupervised learning task that groups sets of items based on their features. It identifies which items are more similar to each other than other items. This might be useful in scenarios such as organizing news articles into groups based on their topics, segmenting users based on their shopping habits, and grouping viewers based on their taste in movies. 
+
+ML.NET 0.2 exposes `KMeansPlusPlusClusterer` which implments K-Means++ clustering. [This test](https://github.com/dotnet/machinelearning/blob/78810563616f3fcb0b63eb8a50b8b2e62d9d65fc/test/Microsoft.ML.Tests/Scenarios/ClusteringTests.cs) shows how to use it (from [#222](https://github.com/dotnet/machinelearning/pull/222)).
+
+#### Train using data objects in addition to loading data from a file: `CollectionDataSource`
+
+ML.NET 0.1 enabled loading data from a delimited text file. `CollectionDataSource` in ML.NET 0.2 adds the ability to use a collection of objects as the input to a `LearningPipeline`. See sample usage [here](https://github.com/dotnet/machinelearning/blob/78810563616f3fcb0b63eb8a50b8b2e62d9d65fc/test/Microsoft.ML.Tests/CollectionDataSourceTests.cs#L133) (from [#106](https://github.com/dotnet/machinelearning/pull/106)). 
+
+#### Easier model validation with cross-validation and train-test
+
+[Cross-validation](https://en.wikipedia.org/wiki/Cross-validation_(statistics)) is an approach to validating how well your model statistically performs. It does not require a separate test dataset, but rather uses your training data to test your model (it partitions the data so different data is used for training and testing, and it does this multiple times). [Here](https://github.com/dotnet/machinelearning/blob/78810563616f3fcb0b63eb8a50b8b2e62d9d65fc/test/Microsoft.ML.Tests/Scenarios/SentimentPredictionTests.cs#L51) is an example for doing cross-validation (from [#212](https://github.com/dotnet/machinelearning/pull/212)).
```

Comment:
Could we please wordwrap these things? One of the nice things about Markdown is that it is simultaneously both interpretable both as a "rendered" document, plus as its actual raw form... hence the deliberate choice of the name "Markdown" as opposed to "Markup." Whenever we don't properly wordwrap these things, that compromises that crucial advantage it has.

There is some language I wrote earlier in some other contexts, about writing documentation. You may find it useful, or not.

-----------------------------

One of the things I would suggest is installing text wrapping. At the time of
writing Visual Studio Code does not have hard wrapping installed by default,
but there is a nice extension called `rewrap`.

Hit *Control-p*, then type the command `ext install rewrap`. You will be able
to install rewrap from here. Once you do, you can change its settings: in
particular I would change its wrapping column to 78, in order to reflect our
guidelines. This is controlled through the setting
`"rewrap.wrappingColumn": 78`.

Visual Studio Code uses a JSON format configuration file (somewhat
akin to, say, Sublime Text and some other editors), and you can configure the
behavior of plugin architecture here as well. So if you go to the menu "File,"
then "Preferences," then "Settings," you will your user configuration. If
you've never provided settings, your user settings will be empty, looking
something like this:

```javascript
// Place your settings in this file to overwrite the default settings
{
}
```

Once you change it, it will look something like this.

```javascript
// Place your settings in this file to overwrite the default settings
{
    "rewrap.wrappingColumn": 78
}
```

Thenceforth, whenever you want to wrap a paragraph, Alt-Q will do so.

Reply:
Addressed.